bootstrap-colequalizer.js

```diff
@@ -0,0 +1,106 @@
+/* global window, module, define, jQuery, require */
+/*
+ * Bootstrap ColEqualizer v1.0.0
+ * https://github.com/gsmke/bootstrap-colequalizer
+ */
+
+;(function(factory) {
+    'use strict';
+
+    if (typeof define === 'function' && define.amd) {
+        define(['jquery'], factory);
+    } else if (typeof exports !== 'undefined') {
+        module.exports = factory(require('jquery'));
+    } else {
+        factory(jQuery);
+    }
+
+}(function($) {
+    'use strict';
+
+    var ColEqualizer = window.ColEqualizer || {};
+
+    ColEqualizer = (function () {
+
+        function ColEqualizer(element) {
+
+            var _ = this;
+
+            _.el = element;
+
+            _.colHeight(_.el);
+
+            _.winLoad();
+
+        }
+
+        return ColEqualizer;
+
+    }());
+
+    ColEqualizer.prototype.colHeight = function(element) {
+
+        var _ = this;
+
+        $(element).each(function (index,el) {
+            _.colReset(el);
+            var tallest = 0;
+            $('[class*=col-]',el).each(function (i,e) {
+                var testHeight = $(e).height();
+                if (testHeight > tallest) {
+                    tallest = testHeight;
+                }
+            });
+            $('[class*=col-]',el).height(tallest);
+        });
```

Comment:
This function compares all the heights of the elements and sets them to the tallest one.